deltametrics/mobility.py

```diff
@@ -46,6 +46,10 @@ def check_inputs(chmap, basevalues, time_window, landmap=None):
         chmap = chmap.mask
```

Comment:
Ah yeah, I see that now, I've changed the statement in here so that any subclass of `BaseMask` as input will raise an error:
```
TypeError: Cannot input a Mask directly to mobility metrics. Use a list-of-masks instead.
```

Reply:
also added a test to check this.